api/v1alpha1/podhttpchaos_types.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// PodHttpChaosSpec defines the desired state of IoChaos
+type PodHttpChaosSpec struct {
+	// ProxyPorts represents the target ports to be proxy of.
+	ProxyPorts []int32 `json:"proxy_ports"`
+
+	// Pid represents a running tproxy process id
+	// +optional
+	Pid int64 `json:"pid,omitempty"`
+
+	// StartTime represents the start time of a tproxy process
+	// +optional
+	StartTime int64 `json:"start_time,omitempty"`
```

Comment:
why need to know the start time of a tproxy process?

Reply:
Because the PID may be occupied by other processes if the tproxy exited by accident.